gengo/Cargo.toml

```
@@ -18,7 +18,7 @@ max-performance = ["gix/max-performance"]
 max-performance-safe = ["gix/max-performance-safe"]
 
 [dependencies]
-gix = { version = ">= 0.56, <= 0.69", default-features = false, features = [
+gix = { version = ">= 0.56, <= 0.71", default-features = false, features = [
```

```suggestion
gix = { version = ">= 0.56, <= 0.70", default-features = false, features = [
```
The highest known working version is 0.70 (`<=`, not `<`).

---

That’s a typo indeed; I meant `< 0.71`, but your suggestion should be fine too. Thanks.